homeassistant/components/whirlpool/climate.py

```diff
@@ -93,6 +93,7 @@ class AirConEntity(ClimateEntity):
 
     _attr_fan_modes = SUPPORTED_FAN_MODES
     _attr_has_entity_name = True
+    _attr_name = None
```

Comment:
Nice!

There are probably other integrations which set `_attr_has_entity_name` to True but don't set a name, we need to update those to explicitly set the name to None as outlined in this blog post: https://developers.home-assistant.io/blog/2023/06/28/entity-name-changes

Reply:
I skimmed all config flow integrations and I found a lot with just running unit tests. I also found a few with just looking at the entities. So I don't think we're at 100%, but I think we're close.